From 172560b845460b6121154f88221c855542219943 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date: Wed, 15 May 2019 22:39:02 +0200
Subject: [PATCH] Fix build with exiv2-0.27.1
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Summary:
Upstream is moving things around between headers. While they are busy doing that, they recommend including exiv2.hpp.

```
lib/imagemetainfomodel.cpp:293:40: error: expected unqualified-id before ‘&’ token
             } catch (const Exiv2::Error& error) {
                                        ^
```

Reviewers: #gwenview, ngraham

Reviewed By: #gwenview, ngraham

Subscribers: ngraham, devurandom

Tags: #gwenview

Differential Revision: https://phabricator.kde.org/D21148
---
 lib/imagemetainfomodel.cpp | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/lib/imagemetainfomodel.cpp b/lib/imagemetainfomodel.cpp
index d7b89ab2..58f1c13d 100644
--- a/lib/imagemetainfomodel.cpp
+++ b/lib/imagemetainfomodel.cpp
@@ -33,9 +33,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #include <KFormat>
 
 // Exiv2
-#include <exiv2/exif.hpp>
-#include <exiv2/image.hpp>
-#include <exiv2/iptc.hpp>
+#include <exiv2/exiv2.hpp>
 
 // Local
 #ifdef HAVE_FITS
